src/ui/Pricing/SubscriptionSettings/index.svelte

```diff
@@ -179,12 +181,56 @@
     gap: 16px;
   }
 
+  .description {
+    margin-top: 6px;
+  }
+
   :global(.phone),
   :global(.phone-xs) {
     .btn-1 {
       width: 100%;
       margin-top: 8px;
     }
+
+    .card {
+      --margin: 6px;
+    }
+
+    .setting {
+      display: flex;
+      flex-direction: column;
+      align-items: flex-start;
+    }
+
+    .description {
+      margin-top: 6px;
+    }
+
+    .cancel-btn {
+      margin-top: 12px;
+    }
+
+    .txt-right {
+      margin-top: 12px;
+      display: flex;
+      align-items: center;
+    }
+
+    .btn-2 {
+      --v-padding: 7px;
+    }
+
+    .nowrap {
+      margin-top: 12px;
+    }
+
+    div.nowrap {
```

Comment:
`nowrap` is utility class and could be used in multiple places within component, hence such style rule will affect wrong elements. Better to assign classes to categorise needed behaviour